.changeset/silly-candles-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': patch
+---
+
+Added the `handleMetaRedirects` transformer in order to support the `mkdocs-redirects` plugin. The `meta` tags generated by `mkdocs-redirects` are allowed in the shadow dom. The transformer handles the `meta` tags by parsing the url from the tag attribute and then executes a redirect via react-router.
```

Comment:
A personal opinion here, but since changelogs are customer-facing, I don't like including too much technical information where it's not relevant (i.e. it's not a public API they can use)... In this case, I would focus more on the behaviour that we now have support for mkdoc-redirects and auto-redirection of old documentation paths

Reply:
Updated the `changeset` to be more behavior focused.